lib/versioned/^3.7.0/index.js

```diff
@@ -55,7 +57,7 @@ function execute(opts, env, config) {
     }
     if (opts.tasksJson) {
       tree = taskTree(gulpInst.tasks);
-      if (config.description && isString(config.description)) {
+      if (config.description && typeof config.description === 'string') {
```

Comment:
Where is `isString` defined?  Was this a bug?

Reply:
Yes, this is a bug.